### PR TITLE
New Templates Added

### LINKED ISSUE #7: feat: add more themes
## Description

Initially, we have 3 templates for certificates. We potentially need 3 more different templates. Kindly share the template before working on it!

### PR DESCRIPTION
**Pull request by CDPhu**
fixes #7 
I have compiled a list of 5 new templates.
@vedant-jain03, Please tell me if I need to change anything within them

### CODE REVIEW

**Comment by vedant-jain03** [comment]
@LostButton This is not the way of adding templates, kindly refer to the current templates implementation. It is way more then just  adding images, let me know if you had doubt!!!

**Review by yashikajotwani12 — CHANGES_REQUESTED** [review]
[@LostButton](https://github.com/LostButton) This is not the way of adding templates, kindly refer to the current templates implementation. It is way more then just adding images, let me know if you have any doubt!!!

**Review by yashikajotwani12 — CHANGES_REQUESTED** [review]
Hey @LostButton, You need to add your templates here.
https://github.com/vedant-jain03/certificate-generator/blob/master/src/components/Homepage.jsx#L156
I did tried adding and I saw this where the headings were merging, so after adding make sure everything is working.
Screenshot from 2022-10-03 10-17-32
Add and let me know if you need any help, Good work  :+1:

**Comment by CDPhu** [comment]
@yashikajotwani12  The tenplate should be fixed now

**Comment by CDPhu** [comment]
It should be formatted properly now, if not please alert me so i can fix any issues.

**Comment by yashikajotwani12** [comment]
Hey @LostButton, you have not added your templates like it is added here https://github.com/vedant-jain03/certificate-generator/blob/8e3021ac5dd66edc84624dfdaca59895efa14a0e/src/components/Homepage.jsx#L156, please add the code in Homepage.jsx.

**Comment by CDPhu** [comment]
> Hey @LostButton, you have not added your templates like it is added here https://github.com/vedant-jain03/certificate-generator/blob/8e3021ac5dd66edc84624dfdaca59895efa14a0e/src/components/Homepage.jsx#L156, please add the code in Homepage.jsx.

But jsx is already updated. It was updated during my second commit?

**Comment by CDPhu** [comment]
OHHHHH I SEE NOW

**Comment by yashikajotwani12** [comment]
https://user-images.githubusercontent.com/77020164/194121448-fd00dab4-4a15-4297-bb9e-cfb00027b81a.mp4

The template when clicked just blanks out the screen, please do check this.

**Comment by CDPhu** [comment]
Can you please check whether its only template seven this is happening to?

**Comment by CDPhu** [comment]
Of all things, it was a **single space**. Issue should be patched now

**Comment by CDPhu** [comment]
@yashikajotwani12 The issue of the blank page should be patched now

**Comment by yashikajotwani12** [comment]
Also the size of the templates is bigger, you can add a scrolling part so that the size remains the same.
Screenshot from 2022-10-06 08-26-07  
After your changes:
Screenshot from 2022-10-06 08-28-28

File: src/components/Homepage.jsx [hunk 1]
```diff
@@ -8,10 +8,16 @@ import template2 from "../cert_templates/Template2.png";
 import firebase from "../firebase"
 import template3 from "../cert_templates/Template3.png";
 import template4 from "../cert_templates/Template4.jpeg";
+import template5 from "../cert_templates/Template5.png";
+import template6 from "../cert_templates/Template6.png";
+import template7 from "../cert_templates/Template7.png";
+import template8 from "../cert_templates/Template8.png";
+import template9 from "../cert_templates/Template9.png";
```
Comment (vedant-jain03) [inline src/components/Homepage.jsx]:
```suggestion
import template9 from "../cert_templates/Template9.gif";
```

It should be `.gif`

File: src/components/Homepage.jsx [hunk 2]
```diff
@@ -83,6 +89,114 @@ export class ComponentToPrint extends React.PureComponent {
...
+                            <h2 style={{ fontSize: '12px', color: 'red', textDecoration: 'underline' }}>Course Director</h2>
+                            <h1 style={{ fontSize: '20px', color: 'black',}}>{this.props.author === '' ? 'Author Name' : this.props.author}</h1>
+                        </div>
+                        {this.props.logo === '' ? '' : <img src={this.props.logo} style={{ position: 'absolute', width: '4rem', borderRadius: '50%', top: '10%', left: '45%' }} alt="logo" />}
+        
+                    </div>
+                );
+            }
+
+            case " template8":{
```
Comment (vedant-jain03) [inline src/components/Homepage.jsx]:
```suggestion
            case "template8":{
```
This extra space is making the template 8 broken.

File: src/components/Homepage.jsx [hunk 3]
```diff
@@ -83,6 +89,114 @@ export class ComponentToPrint extends React.PureComponent {
...
+                            <h2 style={{color: '#0edee6', textTransform:'uppercase', letterSpacing:'3px' , marginBottom: '1rem' }}>{this.props.heading === '' ? 'Certificate of Achievement' : this.props.heading}</h2>
+                            <h3 style={{ fontSize: '16px', fontWeight: '500', color: 'rgb(6, 124, 214)', textTransform: 'uppercase', letterSpacing: '3px' }}>is hereby awarded to</h3>
+                            <h1 style={{ fontSize: '3rem', color: '#0edee6' }}>{this.props.name === '' ? 'Name' : this.props.name}</h1>
+                            <p style={{ fontSize: '15px', fontWeight: '600', color: 'rgb(6, 124, 214)',width:'60%',textAlign:'center' }}>{this.props.desc === '' ? 'for the active participation in the event and for giving efforts,ideas and Knowledge.' : this.props.desc}</p>
+                        </div>
+                        <div className="author" style={{ position: 'absolute', top: '70%', left: '41%', display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', textAlign: 'center' }}>
```
Comment (vedant-jain03) [inline src/components/Homepage.jsx]:
```suggestion
                        <div className="author" style={{ position: 'absolute', top: '68%', left: '44%', display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', textAlign: 'center' }}>
```

This look more centered and clean. WDYT?

File: src/components/Homepage.jsx [hunk 4]
```diff
@@ -83,6 +89,114 @@ export class ComponentToPrint extends React.PureComponent {
...
+                            <h3 style={{ fontSize: '16px', fontWeight: '500', color: 'rgb(174, 74, 255)', textTransform: 'uppercase', letterSpacing: '3px' }}>is hereby awarded to</h3>
+                            <h1 style={{ fontSize: '3rem', color: '#ffffff' }}>{this.props.name === '' ? 'Name' : this.props.name}</h1>
+                            <p style={{ fontSize: '15px', fontWeight: '600', color: 'rgb(255 251 240)',width:'60%',textAlign:'center' }}>{this.props.desc === '' ? 'for the active participation in the event and for giving efforts,ideas and Knowledge.' : this.props.desc}</p>
+                        </div>
+                        <div className="author" style={{ position: 'absolute', top: '70%', left: '41%', display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', textAlign: 'center' }}>
+                            <h2 style={{ fontSize: '12px', color: 'white', textDecoration: 'underline' }}>Course Director</h2>
+                            <h1 style={{ fontSize: '20px', color: 'rgb(231, 133, 58)',}}>{this.props.author === '' ? 'Author Name' : this.props.author}</h1>
```
Comment (vedant-jain03) [inline src/components/Homepage.jsx]:
```suggestion
                            <h1 style={{ fontSize: '20px', color: '#ffff58', fontWeight: '100'}}>{this.props.author === '' ? 'Author Name' : this.props.author}</h1>
```
It looks more decent, WDYT?

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
You are almost there, here are some changes you need to take care of and then we are finally merging this. Great Work @LostButton 💯 

Also you can make scrolling effect by raising new issues. That don't need to add in this PR. WDYT @yashikajotwani12 ?

**Comment by vedant-jain03** [comment]
> Also the size of the templates is bigger, you can add a scrolling part so that the size remains the same. Screenshot from 2022-10-06 08-26-07 After your changes: Screenshot from 2022-10-06 08-28-28

Could you please add a new issue for this after we merge this PR, the context of the PR was just to add templates.
WDYT @yashikajotwani12 ?

File: src/components/Homepage.jsx (hunk 2 above)
Comment (vedant-jain03) [reply src/components/Homepage.jsx]: This extra space is also in line `176`

**Comment by CDPhu** [comment]
@vedant-jain03 Is this ok?

**Comment by yashikajotwani12** [comment]
Hey @LostButton, As suggested by @vedant-jain03  we would open a new issue for the scrolling, but the one thing that I checked is this, do you see the same?

https://user-images.githubusercontent.com/77020164/194330731-05f090eb-5dce-415e-bc0e-46e42d91860c.mp4

Else Everything is fine.

**Comment by CDPhu** [comment]
@yashikajotwani12 The issue should be fixed. a small redirection error but it should be fixed

**Comment by yashikajotwani12** [comment]
Yeah! It is fixed @LostButton, It LGTM.
Good Work :+1:

**Review by yashikajotwani12 — APPROVED** [review]

**Review by vedant-jain03 — APPROVED** [review]
@LostButton Great work, 💯 
Thanks for your valuable contribution!

**Pull request merged**